pkg/kubelet/kubelet.go

```diff
@@ -1049,9 +1128,19 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, dockerContainers dockertools.Docke
 				return err
 			}
 		}
+		podStatus, err = kl.GetPodStatus(podFullName, uid)
```

Comment:
Don't we already do this in getPodInfraContainer? Maybe we can remove it from there and have it in one place after this if statement.